src/redux/missions/missionsSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import MissionService from '../../services/missionsService';
+
+// Actions
+const FETCHED_MISSONS = 'space-travelers/missions/FETCHED_MISSONS';
```

Comment:
[Optional]
Maybe you can try to make actions as a verb for example: `FETCH_MISSION` instead of FETCHED_MISSONS.
NB: Pay attention to the typo: you forgot to add `i` in _Missons_